get_id.js

```diff
@@ -203,7 +203,7 @@ function hideblocks() {
     //     console.log(name + " placeholder undefined")
     //     // $(`[name=${name}]`).val(placeholder)
     // }
-    for (let i = 2; i < 44; i++) { // выведет 0, затем 1, затем 2
+    for (let i = 2; i < 45; i++) { // выведет 0, затем 1, затем 2

```

Comment:
WTF????
If numbers of blocks increase you need to fix code!!!

There is cool code with jQuery:
$('figure', {elem of form}).hode():

Didi you see code of httpgo?